src/my_react/render.ts

```diff
@@ -0,0 +1,85 @@
+const addAttribute = ($elem: HTMLElement, props: JSX.Props, prop: string) => {
+  /* props의 종류에 따라 필요한 처리, 예외 케이스 더 확인되면 추가 필요 */
+  switch (prop) {
+    case 'styles': {
+      Object.keys(props[prop]).forEach((key) => {
+        ($elem.style as any)[key] = props[prop][key];
+      });
+      return;
+    }
+    case 'key':
+      return;
+    case 'children':
+      return;
+    default: {
+      prop.slice(0, 2) === 'on'
```

Comment:
```suggestion
      prop.startsWith("on")
```
`startsWith`으로 접두사 체크하는 메서드도 있어서 이것도 괜찮은 것 같아요!